static/src/stylesheets/module/content/_content.scss

```diff
@@ -778,6 +778,19 @@
     }
 }
 
+.button--small .inline-twitter-bird svg {
+    height: 12px;
+    width: 14px;
+}
+
+.button--small .inline-twitter-bird.i-left {
```

Comment:
These selectors seem very specific and the the numbers a bit magic, is there a way of doing this in a more general way? Just my 2p.